index.html

```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Mein Profil</title>
   </head>
+  <h1>Das ist unsere Profil Seite</h1>
   <body></body>
```

Comment:
```suggestion
<body>
  <h1>Das ist unsere Profil Seite</h1>
  </body>
```
So sollte es sein...